airbyte-integrations/connectors/source-freshsales/source_freshsales/schemas/open_tasks.json

```diff
@@ -16,6 +17,9 @@
     "created_at": { "type": ["null", "string"] },
     "updated_at": { "type": ["null", "string"] },
     "outcome_id": { "type": ["null", "integer"] },
-    "task_type_id": { "type": ["null", "integer"] }
+    "task_type_id": { "type": ["null", "integer"] },
+    "updater_id": { "type": ["null", "integer"] },
+    "targetables": { "type": ["null", "string"] },
+    "completed_date": { "type": ["null", "string"] }
```

Comment:
I think out of scope for this PR, and a question rather than a suggestion: in the future would be be able to use `airbyte_type`s here to be distinct about the date format?

Reply:
My best guess is `Yes`, but still worth double-checking.